### ISSUE TITLE
Is this project maintained?

### ISSUE THREAD

**Issue by MaddyGuthridge** [cenkalti/pyhtml#18]
Hello! I am a teacher for a university course (COMP1010 at UNSW) that uses PyHTML to help teach basic web servers and HTML. We're currently in discussions about the future of PyHTML in our course, so would appreciate some insight into the state of the project. More specifically, we have the following questions:

* Are you planning on fixing existing bugs (such as #17), as well as any new bug reports the may occur? We want to be sure that the tools we use remain stable so that students can focus on their own work, rather than being blocked by issues with tools we teach.

* If we create merge requests to fix bugs or add missing tags, can we expect new releases to be published quickly? Slow response times may lead to frustration if they cause students to miss deadlines.

* If we create merge requests to improve or refactor the project (eg to improve intellisense support and type safety in editors), can we expect these to be merged, or will you only accept small improvements? We hope to incrementally build these features to improve accessibility for students and tutors.

Of course, I don't want to push you into maintaining an old project that you're not passionate about, so please don't feel any pressure to do additional work on it if you don't want to. We just want some insight into the state of the project so that we can gauge how/if it should be included in the course into the future. We may also consider maintaining a fork of the project if we decide it will be beneficial to the course.

Thanks for your work on this project!

**Comment by cenkalti** [comment]
Hi @MiguelGuthridge ,

I'm glad you're using PyHTML to teach about web servers and HTML.

- Yes, I am willing to fix bugs. 
- I can publish new versions in 24-48 hours after a change is merged.
- Yes, I am willing to merge small improvements but cannot give guarantee for big refactors. I would like to keep the code readable and maintainable so it's hard for me to say anything before seeing the actual PRs.

As I am not actively using this package in any of my projects currently, I am not planning to have significant changes. It's pretty much in "keep the lights on" state but it's all right if it would be beneficial to you in any way.

Please let me know if you have other questions.

**Comment by MiguelGuthridge** [comment]
Awesome! Thanks for the info!

**Comment by MiguelGuthridge** [comment]
Hey, @cenkalti !

Thanks again for making such an awesome project! We decided that the scope of our desired changes was too great to do through the original project, so I've created [`pyhtml-enhanced`](https://github.com/COMP1010UNSW/pyhtml-enhanced), which is a rewrite that offers a few useful features (primarily type safety and documentation for all HTML tags).

I just wanted to thank you for the inspiration your project brought while I was working!
